infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -188,7 +191,15 @@ public ServerContext(Map<String, Object> serverConfig) {
         // Metrics setup & reporting configuration
         if (!isMetricsReportingSetUp(metrics)) {
             MetricsUtils.metricsReportingSetup(metrics);
+            logMetricsSize();
         }
+
+        FileChannelPerf.setupParameters(OP_SPIKE_CNT, OP_SPIKE_LATENCY, false);
+        logMetricsSize ();
+    }
+
+    public void logMetricsSize() {
+        log.debug("corfu server metrics size " + MetricsUtils.sizeOf.deepSizeOf(ServerContext.getMetrics ()));
```

Comment:
-> 
```
log.debug("corfu server metrics size {}", MetricsUtils.sizeOf.deepSizeOf(ServerContext.getMetrics()));
```

Reply:
It is a debug message to know the space used by metrics.

Reply:
yes, just use it properly. Here is the right one:
```
log.debug("corfu server metrics size {}", MetricsUtils.sizeOf.deepSizeOf(ServerContext.getMetrics()));
```

Reply:
BTW, because `deepSizeOf` can be very slow operation you must use following code:
```
if(log.isDebugEnabled()) {
  log.debug("corfu server metrics size {}", MetricsUtils.sizeOf.deepSizeOf(ServerContext.getMetrics()));
}
```